app/services/api/index.ts

```diff
@@ -22,6 +25,14 @@ class Api {
     this.client.setHeader('Authorization', `Bearer ${token}`);
   };
 
+  getCourse = (year = 19) => {
+    return this.client.get(`/exchange_rates?json&date=01.01.20${year}}`);
```

Comment:
I think you should take a full year (2015, 2016, 2017) etc. not just last numbers